apps/web/src/utils/site/getSiteSettings.ts

```diff
@@ -1,8 +1,17 @@
-import { FooterLink } from '@eventuras/ui/Footer';
 import { Logger } from '@eventuras/utils';
 
 import Environment from '@/utils/Environment';
 
+interface FooterProps {
+  siteTitle?: string;
+  links?: FooterLink[];
+  publisher?: Publisher;
+}
+export interface FooterLink {
```

Comment:
I think these should be in the Footer file :) perhaps sitesettings should be a Context thing as well, to clean up and prevet these dependencies